assemblies/enabling-ai-safety-with-guardrails.adoc

```diff
@@ -0,0 +1,57 @@
+:_module-type: ASSEMBLY
+
+ifdef::context[:parent-context: {context}]
+[id="enabling-ai-safety-with-guardrails_{context}"]
+= Enabling AI safety with Guardrails
+
+The TrustyAI Guardrails Orchestrator service is a tool to invoke detections on text generation inputs and outputs, as well as standalone detections.
+
+It is underpinned by the open-source project link:https://github.com/foundation-model-stack/fms-guardrails-orchestrator[FMS-Guardrails Orchestrator] from IBM. You can deploy the Guardrails Orchestrator service through a Custom Resource Definition (CRD) that is managed by the TrustyAI Operator.
+
+The following sections describe the Guardrails components, how to deploy them and provide example use cases of how to protect your AI applications using these tools:
```

Comment:
I'm not sure I understand the structure of this section-  it mixes together component definitions (orchestrator, detectors, etc) with verbs (configure the guardrails gateway, monitor user inputs, enable telemetry)